custom_model_runner/datarobot_drum/custom_task_interfaces/custom_task_interface.py

```diff
@@ -3,7 +3,7 @@
 
 
 class Serializable(object):
-    default_filename = "artifact.pkl"
+    default_artifact_filename = "artifact.pkl"
```

Comment:
We should change the filename to something like `drum_artifact.pkl`, something that users probably wont overwrite